mitosheet/src/components/endo/GridData.tsx

```diff
@@ -63,15 +65,20 @@ const GridData = (props: {
                                 conditionalFormat.backgroundColor = hexToRGB(conditionalFormat.backgroundColor, .4)
                             }
 
-                            if (cellData === undefined || columnDtype == undefined) {
+                            if (cellData === undefined || columnDtype === undefined || columnHeader === undefined) {
                                 return null;
                             }
 
+                            const isColumnCreated = reconIsColumnCreated(columnHeader, props.uiState, sheetData)
+                            const isColumnModified = reconIsColumnModified(columnHeader, props.uiState, sheetData)
+
                             const className = classNames('mito-grid-cell', 'text-unselectable', {
                                 'mito-grid-cell-selected': cellIsSelected,
                                 'mito-grid-cell-conditional-format-background-color': conditionalFormat?.backgroundColor !== undefined,
                                 'mito-grid-cell-hidden': props.editorState !== undefined && props.editorState.rowIndex === rowIndex && props.editorState.columnIndex === columnIndex,
-                                'right-align-number-series': isNumberDtype(columnDtype)
+                                'right-align-number-series': isNumberDtype(columnDtype),
+                                'created-recon-background-color': isColumnCreated,
```

Comment:
So for odd and event, you just need to add two more cases, and then check if it's an odd or even row index. Pretty simple I think.